sdk/appconfiguration/feature-management/README.md

```diff
@@ -0,0 +1,236 @@
+1. Twenty percent of a group known as "Ring1" users are included in the Beta.
+1. The number of "Ring1" users included in the beta is bumped up to 100 percent.
+1. Five percent of the user base is included in the beta.
+1. The rollout percentage is bumped up to 100 percent and the feature is completely rolled out.
+1. This strategy for rolling out a feature is built in to the library through the included Microsoft.Targeting feature filter.
+
+##### Defining a Targeting Feature Filter
+
+The Targeting Filter enables a feature flag based on a list of users, groups, or rollout percentages. It has a parameter:
+
+- `Audience` - The audience of the feature flag.
+
+Which contains the following fields:
+
+- `Users` - A list of users that the feature flag is enabled for.
+- `Groups` - A list of groups that the feature flag is enabled for, at different percentages.
+- `DefaultRolloutPercentage` - A percentage value that the feature flag is enabled for.
+- `Exclusions` - An object that contains a list of users and groups that the feature flag is disabled for.
+
+```json
+{
+    "name": "Microsoft.TargetingFilter",
+    "parameters": {
+        "Users": ["user1", "user2"],
+        "Groups": [
+            {
+                "Name": "group1",
+                "RolloutPercentage": 100
+            }
+        ],
+        "DefaultRolloutPercentage": 50,
+        "Exclusion": {
+            "Users": ["user3"],
+            "Groups": ["group2"]
+        }
+    }
+}
+```
+
+The `Users` field is a list of strings that represent the users that the feature flag is enabled for. The `Groups` field is a list of objects that represent the groups that the feature flag is enabled for. The `Name` field is a string that represents the name of the group. The `RolloutPercentage` field is a number between 0 and 100 that represents the percentage of users in the group that the feature flag is enabled for. The `DefaultRolloutPercentage` field is a number between 0 and 100 that represents the percentage of users that the feature flag is enabled for if they are not in any of the groups. The `Exclusions` field is an object that contains a list of users and groups that the feature flag is disabled for. The `Users` field is a list of strings that represent the users that the feature flag is disabled for. The `Groups` field is a list of strings that represent the groups that the feature flag is disabled for.
+
+#### Custom Filters
+
+You can also create your own feature filters by implementing the `FeatureFilter` interface.
+
+```python
+class CustomFilter(FeatureFilter):
+    def evaluate(self, context, **kwargs):
+        ...
+        return True
+```
+
+The `evaluate` method is called when checking if a feature flag is enabled. The `context` parameter contains information about the feature filter from the `parameters` field of the feature filter. Any additional parameters can be passed in as keyword arguments when calling `is_enabled`.
```

Comment:
>  Any additional parameters can be passed in as keyword arguments when calling `is_enabled`.

Did you mean - by calling `evaluate`?

Reply:
`evaluate` is the method named that `FeatureFilter`'s have. See `_featurefilters.py`